File: qc-core/src/main/java/com/eager/questioncloud/core/domain/creator/dto/CreatorInformation.java
package com.eager.questioncloud.core.domain.creator.dto;

import com.eager.questioncloud.core.domain.question.vo.Subject;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class CreatorInformation {
    private Long creatorId;
    private String name;
    private String profileImage;
    private Subject mainSubject;
    private String email;
    private int salesCount;
    private Double rate;
    private String introduction;
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/creator/dto/CreatorInformation.kt
package com.eager.questioncloud.core.domain.creator.dto

import com.eager.questioncloud.core.domain.question.enums.Subject

class CreatorInformation(
    val creatorId: Long,
    val name: String,
    val profileImage: String,
    val mainSubject: Subject,
    val email: String,
    val salesCount: Int,
    val rate: Double,
    val introduction: String,
) {
}
